packages/runtime/lib/errors.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   FailedToRetrieveMetaError: createError(`${ERROR_PREFIX}_FAILED_TO_RETRIEVE_META`, 'Failed to retrieve metadata for service with id "%s": %s'),
   FailedToRetrieveMetricsError: createError(`${ERROR_PREFIX}_FAILED_TO_RETRIEVE_METRICS`, 'Failed to retrieve metrics for service with id "%s": %s'),
   FailedToRetrieveHealthError: createError(`${ERROR_PREFIX}_FAILED_TO_RETRIEVE_HEALTH`, 'Failed to retrieve health for service with id "%s": %s'),
+  FailedToRetrieveCustomHealthcheckError: createError(`${ERROR_PREFIX}_FAILED_TO_RETRIEVE_CUSTOM_HEALTHCHECK`, 'Failed to retrieve custom healthcheck for service with id "%s": %s'),
```

Comment:
Two things here: the spelling of the "c" in "check", and the naming. Rather than "retrieve" I would use the word "execute" or "perform".